### DISCUSSION TITLE
OpenRadioss vs Radioss

### DISCUSSION THREAD

**Discussion by dennyxx82** [OpenRadioss/OpenRadioss#1576] [Users]
what is the difference between radioss and openradioss?

**Comment by MarianBulla** [comment]
Hello dennyxx82,
in general, there is no difference between radioss and OpenRadioss.
OpenRadioss is the mainline development version, where the is continous development.
Based on OpenRadioss there are forcs which will undergo some external validations and QA, which will become Altair Radioss, few times per year.
One major difference is in the encryption routines, which are not available in OpenRadioss, only in Altair Radioss, to protect the existing, industrial used models, e.g. commercial dummies.
There are also some other minor differences, but usually not so important for most of the usecases, e.g. implicite functions, which are embedded in Altair Radioss, but not in OpenRadioss, but the user can add them manually (see the other thread within the OpenRadioss discussions forum).
Hope it helps you ?
Thanks and best regards,
Marian

**Reply by dennyxx82** [reply]
thanks I would like to use this software to simulate a rollover test of a bus, as described by the standard UNECE R66. however I must first validate the model used by the Italian Ministry of Transport

**Reply by MarianBulla** [reply]
Yes, bus rollover is possible, with OpenRadioss. 
I did a presentation on that topic several years ago, at carhs. And some collegues have also experience on modelling it.
You can shorten the calculation time, by switching the entire structure to rigid and release just short before impact to deformable.
There is on example (the bicycle model) showing, how to model it.
Hope it helps you ?
Thanks and best regards,
Marian

**Reply by MarianBulla** [reply]
Here some older slides, showing the needed keywords (in this case for a car, but for bus it is similar):
![image](https://github.com/OpenRadioss/OpenRadioss/assets/105412502/ca60ca99-16c3-4267-9875-d8a178e95dff)
![image](https://github.com/OpenRadioss/OpenRadioss/assets/105412502/b44f3d32-4acb-41ba-a96d-629dadc70c90)

**Reply by dennyxx82** [reply]
great, by any chance do you have a link where I can download the example you shot me? Will he be in Aachen on the 27th, will you be there so you can talk about it in person?
